woeip/apps/data_collection/models.py

```diff
@@ -0,0 +1,118 @@
+from django.contrib.gis.db.models import LineStringField, PointField
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class Organization(models.Model):
+    """An organization, e.g., WOEIP, OpenOakland"""
+    name = models.CharField(max_length=256)
+    website = models.CharField(max_length=256)
+    email = models.CharField(max_length=256)
+    contact = models.ForeignKey('Participant',
+                                related_name='contact', related_query_name='contact',
+                                on_delete=models.SET_NULL, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.name} <{self.email}>"
+
+
+class RoleEnum(models.Model):
+    """Create, read, update, delete permissions"""
+    name = models.CharField(max_length=256)
+    can_create = models.BooleanField()
+    can_read = models.BooleanField()
+    can_update = models.BooleanField()
+    can_delete = models.BooleanField()
+
+    def __str__(self):
+        crud = []
+        if self.can_create:
+            crud.append('create')
+        if self.can_read:
+            crud.append('read')
+        if self.can_update:
+            crud.append('update')
+        if self.can_delete:
+            crud.append('delete')
+
+        crud = '/'.join(crud)
+
+        return f"{self.name} ({crud})"
+
+
+class Participant(models.Model):
+    name = models.CharField(max_length=256)
+    email = models.CharField(max_length=256)
+    organization = models.ForeignKey(Organization, on_delete=models.SET_NULL, blank=True, null=True)
+    role = models.ForeignKey(RoleEnum, on_delete=models.SET_NULL, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.name} <{self.email}> {self.organization} {self.role}"
+
+
+class RouteEnum(models.Model):
+    """Named routes"""
+    name = models.CharField(max_length=256)
+    latlon = LineStringField()
+
+    def __str__(self):
+        return self.name
+
+
+class Session(models.Model):
+    """A single air quality outing. Can link to several SessionData, e.g., raw data files."""
+    date_collected = models.DateTimeField(auto_now_add=True)
+    route = models.ForeignKey(RouteEnum, on_delete=models.SET_NULL, blank=True, null=True)
+    collected_by = models.ForeignKey(Participant, on_delete=models.CASCADE)
+    metadata = JSONField(blank=True, null=True)
```

Comment:
What metadata do you expect to go in this field? Can/should those fields be on the model itself? My YAGNI sense is going off here.